unit-test/background/metrics.js

```diff
@@ -0,0 +1,84 @@
+import { RefreshMetric } from '../../shared/js/background/metrics';
+
+class MockAbnExperimentMetrics {
+    onMetricTriggered() {}
+}
+
+class MockTabTracking extends EventTarget {
+    mockRefreshEvent(tabId, timeStamp = Date.now()) {
+        this.dispatchEvent(
+            new CustomEvent('tabRefresh', {
+                detail: {
+                    tabId,
+                    timeStamp,
+                },
+            }),
+        );
+    }
+}
+
+fdescribe('RefreshMetric', () => {
```

Comment:
I've not seen `fdescribe` before, how's that different from `describe`?

Reply:
Oops - that tells the test running to only run that batch of tests. Removed.